### DISCUSSION TITLE
Document title, author, subject, created

### DISCUSSION THREAD

**Discussion by PetrKaleta** [gettalong/hexapdf#248] [General]
Hello,
is there any way how to set document's title, author, subject and created attributes?

Thank you

**Comment by gettalong** [comment]
Sure, use the `document.trailer.info` dictionary and set the `:Title`, `:Author`, `:Subject` and `:CreationDate` keys appropriately. Also see https://hexapdf.gettalong.org/documentation/api/HexaPDF/Type/Info.html for all available fields.
